src/xmpp/xmpp-im/xmpp_mamtask.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * xmpp_mam.cpp - XEP-0313 Message Archive Management
+ * Copyright (C) 2024 mcneb10
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef XMPP_MAM_TASK_H
+#define XMPP_MAM_TASK_H
+
+#include <QDateTime>
```

Comment:
we prefer to use includes in next order

- current dir
- anyhow local for the project
- system libs
- standard stuff like STL